scrapy/extensions/feedexport.py

```diff
@@ -335,7 +400,7 @@ def _get_uri_params(self, spider, uri_params):
         params = {}
         for k in dir(spider):
             params[k] = getattr(spider, k)
-        ts = datetime.utcnow().replace(microsecond=0).isoformat().replace(':', '-')
+        ts = datetime.utcnow().isoformat().replace(':', '-')
```

Comment:
Any reason for this change?

Reply:
replace(microsecond=0) - removes microseconds. But if several files were sent within one second they will have the same names.